docs/tf-game-token.md

```diff
@@ -0,0 +1,9 @@
+---
+id: tf-game-token
+title: Token Gateway Example
+sidebar_label: Token Gateway Example
+---
+
+ERC20 token example shifting the tokens between chains is located here on [Github](https://github.com/loomnetwork/token-gateway-example)
```

Comment:
im going to merge, but all the docs should be on doc site and GitHub should link back here. Cause otherwise docs on GitHub get outdated